c++/src/ColumnReader.cc

```diff
@@ -993,6 +993,9 @@ namespace orc {
     void nextInternal(ColumnVectorBatch& rowBatch,
                       uint64_t numValues,
                       char *notNull);
+
+    void readArrayIndices(const int64_t *arrayOffsets, uint64_t arrayCount,
```

Comment:
If you don't mind, please put one argument per line.
```cpp
void readArrayIndices(const int64_t *arrayOffsets,
                      uint64_t arrayCount,
```

Reply:
Deleted in latest commit.